### DISCUSSION TITLE
Which node version is recommended?

### DISCUSSION THREAD

**Discussion by yaxu** [knowledgefutures/pubpub#1896] [Support] [answered]
I just tried a `npm install` and `node-gyp` failed to install. It seems this is an incompatibility with node 16 that I have installed, and the package versions in the package-lock.json file. Is there a particular version of node that you use/recommend?

**Comment by gabestein** [comment, marked as answer]
Hello! At the moment, we use 12.13.0 (specified in package.json in the engines section).

I should also mention that, while you're more than welcome to take it for a spin, and we can provide some help here, we're not actively supporting self-hosting at the moment, as PubPub requires access to a large number of third-party services. Over time we'd like to reduce that dependence and make PubPub self-hostable, but for the moment it's not our focus.

More on that: https://github.com/pubpub/pubpub/issues/1125, https://github.com/pubpub/pubpub/discussions/1500
